externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/ExecuteStatement.scala

```diff
@@ -209,6 +218,15 @@ class ExecuteStatement(
     setState(OperationState.FINISHED)
   }
 
+  private def cleanupQueryResult(resultId: String): Unit = {
+    try {
+      executor.cancelQuery(sessionId, resultId)
+    } catch {
+      case _: Throwable =>
+      // ignore further exceptions for cleanup
```

Comment:
Any exception could be logged?

Reply:
I thought of logging the exception, but Flink SQL Client ignores it for some reason (maybe to reduce the noises), so I keep aligned with Flink SQL Client. Either way is okay to me.

Reply:
@link3280, I don't think that this should keep aligned with Flink SQL Client. We could log here for the cancel exception so that the users could know canceling query cause the exception which may not cancel successfully.

Reply:
+1 for sharing more information makes sense

Reply:
@SteNicholas @yanghua Sorry for the late reply. Fixed.